src/creep.prototype.js

```diff
@@ -517,7 +531,7 @@ export function moveTo({
     state.cpu = _.round(cpuUsed + state.cpu);
     if (state.cpu > REPORT_CPU_THRESHOLD) {
       // see note at end of file for more info on this
-      //console.log(`moveTo: heavy cpu use: ${creep.name}, cpu: ${state.cpu} origin: ${creep.pos}, dest: ${destination}`);
+      console.log(`moveTo: heavy cpu use: ${creep.name}, cpu: ${state.cpu} origin: ${creep.pos}, dest: ${destination}`);
```

Comment:
Turned this back on but bumped limit to 15000.  We actually do want to know about creeps that take 15/20 CPU cycles to compute a route.